src/components/campaigns/CampaignCard.tsx

```diff
@@ -117,7 +119,7 @@ export default function CampaignCard({ campaign }: Props) {
           <div
             style={{ position: 'relative', width: '100%', minHeight: '100%', maxHeight: '100%' }}>
             <Image alt={campaign.title} src={pictureUrl} fill style={{ objectFit: 'contain' }} />
-            {reached >= target ? <SuccessfullCampaignTag /> : ''}
+            {campaignState === 'complete' ? <SuccessfullCampaignTag /> : ''}
```

Comment:
hey, let's compare with CampaignState.complete here for safety

Reply:
I agree, you can also create an enum for CampaignState, so we don't hardcode strings